Add article about Testnet faucet (#2516)
near/docs: docs/smart-contracts/quickstart.md

:::

:::info Testnet tokens

> If you need NEAR tokens or want to add funds to your `testnet` account, use the [Testnet Token Faucet](../tools/faucet.md) to get free tokens to experiment and test while building on NEAR.

Reviewer: ```suggestion
Need some `testnet` tokens? use the [faucet](../tools/faucet.md) to top-up your account
```
